add solution

Reyti opened the pull request:
[DEMO LINK](https://<Reyti>.github.io/layout_landing-page/)

**checklist.md - done.**

etojeDenys reviewed (changes requested):
1. show the number on hover
<img width="388" alt="image" src="https://github.com/user-attachments/assets/98bbe6a2-eb33-4031-9536-6a7c0e8d0c39" />

2. the link looks different
3. phone and mail also should be links
<img width="410" alt="image" src="https://github.com/user-attachments/assets/40a279a6-ca19-41e7-8d72-2a6df06e860d" />

4. disable textarea resizing
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/4596bb17-3abf-4b16-92ba-aea56c2452b5" />

5. add the required attribute to all fields
<img width="792" alt="image" src="https://github.com/user-attachments/assets/e54bf403-378d-4d11-b679-275f7bca9a67" />

Reyti commented:
1. Made it come out on hover 
![image](https://github.com/user-attachments/assets/7e4d7985-0beb-4884-b8ca-8fa7a0d28024)
2. Changed to href="tel:+12345555555"
3. They are links now
4. No more resizing 
![image](https://github.com/user-attachments/assets/958f0dbe-1e05-4e5e-973c-b234542ec0ad)
5. Assigned attributes: (The data-info attribute is for future JS access, which I will try to implement when working on the DOM module.)
![image](https://github.com/user-attachments/assets/fc81e4e6-0f20-4bf3-91d3-7e2328ee7a69)
![image](https://github.com/user-attachments/assets/0cc82604-edc1-42df-9c38-f4cecb559ca2)

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
To improve:
1. The phone number styles looks different as on the design, need to fix it 
<img width="256" alt="image" src="https://github.com/user-attachments/assets/e278a06e-5755-406c-a352-26c0ca98f5eb" />

2. Add cursor pointer for all clickable elements 
<img width="517" alt="image" src="https://github.com/user-attachments/assets/c6d880b9-2d0f-45b4-a015-d494f5f69897" />

3. Make these fields required before submit 
<img width="515" alt="image" src="https://github.com/user-attachments/assets/fd387890-8a2c-4499-8911-3649c44ce00b" />

4. Add hover effects here 
<img width="341" alt="image" src="https://github.com/user-attachments/assets/d265c1e9-11a9-4380-bbab-0ee429df8d99" />

5. The font family should be are same 
<img width="130" alt="image" src="https://github.com/user-attachments/assets/224f4073-fee6-43f2-a8b2-c44d7d7c5601" />

6. Hide the additional phone number on the mobile and tablet versions 
<img width="593" alt="image" src="https://github.com/user-attachments/assets/4b575173-1535-4620-9659-dc6a526e4165" />

7. Need to center the background image on the mobile and tablet
<img width="942" alt="image" src="https://github.com/user-attachments/assets/2d389f06-be8b-49fe-aeaf-47614c6be4ca" />

Reyti commented:
1. Done
2. Added cursor: pointer; for button mixin (the only one s missing)
3. Done 
![image](https://github.com/user-attachments/assets/fe6a149d-76cc-4ec2-b985-d57bcc793923)
4. Added translateX(5px) on hover (could be scale, but I think there's too much of that on the page)
5. They are same on my device.
![image](https://github.com/user-attachments/assets/f1ac0331-9390-4e42-8159-6acc4035dc8b)
6. Done
7. Done

Reyti commented:
1. They were implemented in solution 3; I just hadn't run the deployment. It should be visible now. 
![image](https://github.com/user-attachments/assets/11be487c-f7d4-4846-9922-239b06b6cb64)
2. Also done by reloading the page in the previous solution.
3. There's no need for that to be in svg, it was decided that 2x png will be more reliable for the given use case.

anastasiiavorobiova reviewed (changes requested):
Great work!
- <img width="199" alt="Screenshot 2025-05-22 at 19 33 29" src="https://github.com/user-attachments/assets/88a0a601-406b-4a56-be12-7d70ca2e02d2" />
- Consider removing default outlines
- <img width="470" alt="Screenshot 2025-05-22 at 19 33 24" src="https://github.com/user-attachments/assets/62eba3d4-d231-45b8-8500-1d41f7583404" />
- Consider removing default borders

Reyti commented:
With box-sizing: border-box, the padding (14px 16px) stays within the textarea’s width/height, preventing overflow as intended per design. In the screenshot you provided, the borders are placed within the padding pixels, so they don’t seem to affect anything whatsoever. If I’m wrong, please provide more details; otherwise, approve this request.

etojeDenys reviewed (changes requested):
1. don't forget to implement those steps
<img width="883" alt="image" src="https://github.com/user-attachments/assets/a209e708-0df1-46fe-9f59-79e1bfa03f0f" />

etojeDenys reviewed (approved):
but looks in overall
